arrows/mvg/applets/tests/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(kwiver-test-setup)
+
+add_test(NAME mvg:track_features COMMAND kwiver track-features "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4")
```

Comment:
We should probably change the test name to `mvg:applets:track_features`.

Let's not do this right now.  If the tests pass we can merge this and then make this change in the next PR when you add another test.